src/pages/Viewer.js

```diff
@@ -51,15 +51,17 @@ function Viewer({ code, widgets }) {
   }, []);
 
   return (
-    <Widget
-      src={!code && src}
-      code={code} // prioritize code
-      props={{
-        path: src,
-        ...passProps,
-      }}
-      config={{ redirectMap }}
-    />
+    <div className="bg-white">
```

Comment:
Instead of doing this so "Show post" and "show comment" show -- how about we modify these links to point to the Build DAO gateway style of posts/comments?

This will involve pulling in the full notifications from mob.near so we can modify the details, but that is okay.

Reply:
@itexpert120 yes I think it should all be forked, especially if we will be introducing more notification types already.

We do have notify.near if we'd like this to be a separate project @Megha-Dev-19 
(thinking about more customizable notifications, in how they look and how they act/link)